src/webrtc/callFeed.ts

```diff
@@ -21,7 +21,7 @@ import { RoomMember } from "../models/room-member";
 import { logger } from "../logger";
 import { TypedEventEmitter } from "../models/typed-event-emitter";
 
-const POLLING_INTERVAL = 200; // ms
+const POLLING_INTERVAL = 1; // ms
```

Comment:
I wonder what is the correct value to put here - has there been research done on this?

Reply:
Okay, this change was necessary in order to achieve low latency activation. Before the looper just checked whether or not you are loud enough (and for an extended period) to be considered speaking. 
Now we want instant checks on whether or not the user is loud enough to activate their audio track which needs this low latency change. Otherwise as dbkr mentioned, the first part of a sentence etc will be cut off.

Reply:
Will test, with 10ms small things like snipping fingers will no longer be recognized.

Reply:
Yeah, so I did some more testing, and sometimes it does indeed cut off a very small portion of the start.
I think the only solution here is to A: directly manipulate the audio, which I have no idea how to do. 
or B: introduce a small amount of latency to properly handle the audio before sending. 

Perhaps you have an idea here, sorry.